::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2019 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Mon, 27 Jan 2020 17:10:04 +0100
Subject: [PATCH] region: s/syncdb/migrate/ for MaaS 2.4 compatibility

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 maas/region.sls | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/maas/region.sls b/maas/region.sls
index ca876ee..31e9906 100644
--- a/maas/region.sls
+++ b/maas/region.sls
@@ -186,7 +186,7 @@ maas_region_services:
 maas_region_syncdb:
   cmd.run:
   - names:
-    - maas-region syncdb --noinput
+    - maas-region migrate --noinput
   - require:
     - file: /etc/maas/regiond.conf
   {%- if grains['saltversioninfo'][0] >= 2017 and grains['saltversioninfo'][1] >= 7 %}
